### DISCUSSION TITLE
Deployment issues in asp.net core

### DISCUSSION THREAD

**Discussion by zgbjty** [dotnet/MQTTnet#1752] [Q&A]
In asp.net Core uses WebSockets, because of the use of Kestrel, it is not possible to deploy in IIS, how to set this up, or use other methods to deploy the project to other platforms

I followed the asp net in the sample, using net 6

**Comment by gfoidl** [comment]
For reference, the same question is in https://github.com/dotnet/MQTTnet/issues/1750 (why?)
